### PR TITLE
Witches Heart Recipe

### PR DESCRIPTION
**Pull request by DanFCo**

### CODE REVIEW

File: src/recipes/witches-heart.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "directions": [
+        "Add about 1/2 - 1 tsp of powdered dry ice to the bottom of the glass (optional).",
+        "Place the apple brandy and purple shimmery liqueur in a shaker.",
+        "Add 1 ice cube and shake for a few seconds to chill the drink.",
+        "Strain the drink into a martini glass.",
+        "Top up with more Blackberry liqueur if necessary. ",
+        "Pour 1 tsp of grenadine syrup, about an inch from the surface of the drink -
+        the grenadine should sink to the bottom, creating a bleeding effect.",
+        "Add about 1/2 tsp of powdered dry ice on top and serve with a stirrer,
+        so that your guests can stir the potion to create that shimmery, smoky effect"
```
Comment (alfg) [inline src/recipes/witches-heart.json]: Invalid json due to the line break. You can make this one line.

Comment (DanFCo) [reply src/recipes/witches-heart.json]: Got it! sorry about that

Comment (alfg) [reply src/recipes/witches-heart.json]: No worries! Thanks for fixing.

**Comment by alfg** [comment]
Looks good! Thanks!

**Pull request merged**